src/tokens/icons.ts

```diff
@@ -14,6 +14,7 @@ export const MDI_ICONS = {
   BOOK_LOCATION_OUTLINE: 'mdi:book-location-outline',
   BRIEFCASE_VARIANT_OUTLINE: 'mdi:briefcase-variant-outline',
   CALENDAR_MONTH_OUTLINE: 'mdi:calendar-month-outline',
+  CHART_ALERT: 'mdi:chat-alert',
```

Comment:
```suggestion
  CHAT_ALERT: 'mdi:chat-alert',
```
Et du coup ça doit aller plus bas

Reply:
@elrhourha toujours celui-là